contrib/terraform/upcloud/README.md

```diff
@@ -108,6 +110,8 @@ terraform destroy --var-file cluster-settings.tfvars \
   * `cpu`: number of cpu cores
   * `mem`: memory size in MB
   * `disk_size`: The size of the storage in GB
+  * `force_public_ip`: If `use_public_ips` is set to false, this forces a public NIC on the machine. Useful if you're migrating from public nodes to only private. Defaults to `false`
+  * `force_no_user_data`: If `private_network_dns` is set existing nodes will be deleted. This forces this mahcine to not add the user_data. Useful if you're migrating from public nodes to only private. Defaults to `false`
```

Comment:
Same with this. It's not clear to me _why_ I would need to use this variable.

Reply:
https://github.com/elastisys/kubespray/pull/20/commits/b604976dbc0078406c3922eac642bde08d0365c4